spec/graphql/utilities/searchers_spec.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../../app/graphql/utilities/searchers.rb'
```

Comment:
You should be able to use `Searchers` namespace here instead of manually requiring - you may need to tell Rails to look in different spots to load files though it looks under app by default.

Reply:
I was kind of confused by this as well. It wasn't needed when I ran the whole spec, but was when I ran this individual file.

I guess that is because I have to do what you say, tell rails about the utilities folder.

Reply:
I added the folder to autoload_paths in 5204691. This feels like it should be unnecessary. Maybe an issue with Spring? I tried stopping Spring and trying again but the folder is still not recognised.